CONTRIBUTING.md

```diff
@@ -172,4 +176,4 @@ See the [conventional commits doc](https://www.conventionalcommits.org/en/v1.0.0
 
 ## Pull requests
 
-In order to ensure conventional commits are followed, pull requests will run a check to indicate whether the PR is following the convention or not. The [Semantic Pull Request](https://github.com/probot/semantic-pull-requests) status check will ensure your pull requests are semantic before you merge them.
+In order to ensure conventional commits are followed, pull requests will run a check to indicate whether the PR is following the convention or not. The [Semantic Pull Request](https://github.com/amannn/action-semantic-pull-request) status check will ensure your pull requests are semantic before you merge them.
```

Comment:
Just an idea, but it'd be great to mention somewhere that developers can modify the conventional commit right before merging if additional edits were needed.

Reply:
Good idea, I can just copy and paste what the script tells them on the CLI https://github.com/Esri/calcite-components/blob/5cd7119f6eb575756d5232fd621ae0441b2c0b8f/support/checkSquashMergeableBranch.ts#L27

Reply:
Oh nevermind I misread, yes thats also I good idea lol